proto/api/v1/remote_runner.proto

```diff
@@ -14,6 +14,12 @@ message RunRequest {
   string branch = 2;
   string commit_sha = 3;
 
+  // Any local patches that should be applied to the repo before
+  // running the command. Each patch should be base64-encoded.
+  // Patches will be applied using "git apply" in the root directory of the
+  // repository.
+  repeated string patches = 11;
```

Comment:
nit: use `repeated bytes` here instead of `repeated string`. The JSON encoding is still the same (base64 string - see [docs](https://protobuf.dev/programming-guides/json/)) but if using other encodings such as the wire format (e.g. go-grpc) then `bytes` is usually more convenient. It also means we don't have to manually decode from base64 on the server side.

```suggestion
  // Any local patches that should be applied to the repo before
  // running the command.
  // If using the JSON API, specify patches as base64-encoded strings.
  // Patches will be applied using "git apply" in the root directory of the
  // repository.
  repeated bytes patches = 11;
```

Reply:
I started with that, but found bytes challenging to generate and format correctly in a CURL request from the command line. I was never able to get it to work 😅 It depends how customers generate these API requests. If they're generating programmatically in a script, it's easy to use bytes. I figured for our external API we should use something more convenient

Reply:
Ah nevermind - you mean that grpc will automatically convert base64-encoded strings into bytes. Updated this